diff
Description:
	Basic changes to make to wnd (pywin) to get it to work
	with Python 2.5.

Files:
	//depot/main/Apps/Komodo-open/contrib/pywin/dlgs/choosecolor.py	# edit
	//depot/main/Apps/Komodo-open/contrib/pywin/fwtypes.py	# edit
	//depot/main/Apps/Komodo-open/contrib/pywin/wintypes.py	# edit
	

Differences:

==== //depot/main/Apps/Komodo-open/contrib/pywin/dlgs/choosecolor.py#1 - c:/home\ericp\main\Apps\Komodo-open\contrib\pywin\dlgs\choosecolor.py ====
Index: main/Apps/Komodo-open/contrib/pywin/dlgs/choosecolor.py
--- pywin/dlgs/choosecolor.py.~1~	Tue Aug 28 14:30:37 2007
+++ pywin/dlgs/choosecolor.py	Tue Aug 28 14:30:37 2007
@@ -49,7 +49,7 @@
 						
 			customcolors= kwargs.get('customcolors')
 			if customcolors: 	
-				for x, i in enumerate(customcolors):
+				for i, x in enumerate(customcolors):
 					self._dlgs_colors[i]= x
 			cc.lpCustColors=pointer(self._dlgs_colors)
 			
==== //depot/main/Apps/Komodo-open/contrib/pywin/fwtypes.py#1 - c:/home\ericp\main\Apps\Komodo-open\contrib\pywin\fwtypes.py ====
Index: main/Apps/Komodo-open/contrib/pywin/fwtypes.py
--- pywin/fwtypes.py.~1~	Tue Aug 28 14:30:37 2007
+++ pywin/fwtypes.py	Tue Aug 28 14:30:37 2007
@@ -25,7 +25,6 @@
 												MAKELONG,
 												COPYDATASTRUCT) 
 
-from ctypes.com import GUID
 import sys, atexit
 #::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
 
==== //depot/main/Apps/Komodo-open/contrib/pywin/wintypes.py#1 - c:/home\ericp\main\Apps\Komodo-open\contrib\pywin\wintypes.py ====
Index: main/Apps/Komodo-open/contrib/pywin/wintypes.py
--- pywin/wintypes.py.~1~	Tue Aug 28 14:30:37 2007
+++ pywin/wintypes.py	Tue Aug 28 14:30:37 2007
@@ -3,6 +3,7 @@
 
 import traceback
 from ctypes.wintypes import *
+from ctypes import *
 
 user32 = windll.user32
 gdi32 = windll.gdi32
@@ -310,4 +311,4 @@
 				Found ver %s.%s on your system'''.replace('\t', '')
 	Msg(0, msg % VER_COMCTL, 'wnd-module', 'ok', 'systemmodal')
 
-#OS_NAME = winos.Name()+#OS_NAME = winos.Name()
End of Patch.

